projects/storefrontlib/shared/components/ng-select-a11y/ng-select-a11y.module.ts

```diff
@@ -0,0 +1,8 @@
+import { NgModule } from '@angular/core';
+import { NgSelectA11yDirective } from './ng-select-a11y.directive';
+
+@NgModule({
+  declarations: [NgSelectA11yDirective],
+  exports: [NgSelectA11yDirective],
+})
+export class CxNgSelectA11yModule {}
```

Comment:
Perhaps the class name caan lose the `cx` prefix?

Seems like the prefix only applies to the selector for other directives.